test/celeritas/ext/Vecgeom.test.cc

```diff
@@ -156,39 +156,40 @@ TEST_F(SimpleCmsTest, track)
     auto geo = this->make_geo_track_view({0, 0, 0}, {0, 0, 1});
     EXPECT_EQ("vacuum_tube", this->volume_name(geo));
 
-    auto next = geo.find_next_step(100);
-    EXPECT_SOFT_EQ(100, next.distance);
+    auto next = geo.find_next_step(100 * units::centimeter);
```

Comment:
Can we use here:
   using cm = units::centimeter;
and then use `cm` instead of `units::centimeter` in all the following lines?

Reply:
You must be looking at an old version? Those are all `to_cm(100)` (for example) now.